spaceship/lib/spaceship/tunes/availability.rb

```diff
@@ -15,15 +15,32 @@ class Availability < TunesBase
       # @return (String) App available date in format of "YYYY-MM-DD"
       attr_accessor :app_available_date
 
+      # @return (Bool) app enabled for b2b users
+      attr_accessor :b2b_app_enabled
+
+      # @return (Bool) app enabled for educational discount
+      attr_accessor :educational_discount
+
+      # @return (Bool) b2b available for distribution
+      attr_accessor :b2b_unavailable
+
+      # @return (Array of Spaceship::Tunes::B2bUser objects) A list of users set by user - if not
+      # then the b2b user list that is currently set
+      attr_accessor :b2b_users
+
       attr_mapping(
         'theWorld' => :include_future_territories,
         'preOrder.clearedForPreOrder.value' => :cleared_for_preorder,
-        'preOrder.appAvailableDate.value' => :app_available_date
+        'preOrder.appAvailableDate.value' => :app_available_date,
+        'b2BAppFlagDisabled' => :b2b_unavailable
       )
 
       # Create a new object based on a set of territories.
       # @param territories (Array of String or Spaceship::Tunes::Territory objects): A list of the territories
       # @param params (Hash): Optional parameters (include_future_territories (Bool, default: true) Are future territories included?)
+      # This method has serious implications on vpp + educational discount but i don't want to break
+      # existing support. please be very careful when using this and provide appropriate params.
+      # In future, we should create availability from territories as an instance method not as a class method.
```

Comment:
Can we remove this comment block after the `@param` and move it to a new line between the `Create` and the first `@param` saying something like...
```
Create a new object based on a set of territories.
This will override any values already set for cleared_for_preorder, app_available_date, b2b_unavailable, b2b_app_enabled, and educational_discount
# @param territories (Array of String or Spaceship::Tunes::Territory objects): A list of the territories
# @param params (Hash): Optional parameters (include_future_territories (Bool, default: true) Are future territories included?)
```

Reply:
Done!